src/components/dialog.css

```diff
@@ -24,12 +24,52 @@
  * The maximum width allowed for the dialog is 480 px. Default width
  * is 100% (for container width < 480 px).
  */
-@media (min-width: 480px) {
+@media (max-width: 640px), (max-height: 640px) {
   .swg-dialog,
   .swg-toast {
     width: 480px !important;
     left: -240px !important;
     margin-left: calc(100vw - 100vw / 2) !important;
+    box-sizing: border-box;
```

Comment:
It'd seem the box-sizing should be the same for all variants?

Reply:
Done